src/plugins/ncr/components/slug-editor/index.jsx

```diff
@@ -99,7 +99,13 @@ class SlugEditor extends React.Component {
 
   handleChange({ target: { value: slug } }) {
     this.setState((prevState, { initialSlug }) => ({
-      slug: slug.replace(' ', '-'),
+      slug: slug
+        .replace(/^[-/\s]|[\s]$/g, '')
```

Comment:
this looks good, thanks for adding these, one really tiny change and we are done:

```suggestion
        .replace(/^[-/\s]+|[\s]+$/g, '')
```

allows catching things like `/ /2019/10/16`